docs/quickstart.md

```diff
@@ -62,7 +62,7 @@ vim /etc/rancher/rke2/config.yaml
 ```
 Content for config.yaml:
 ```bash
-url: https://<server>:9345
+server: https://<server ip>:9345
```

Comment:
Cna a hostname be used here as well?

Reply:
yes it can! Do you think "server" is more clear then? Or "server address" maybe?

Reply:
server should suffice.